src/components/LoginPage/Slide3.js

```diff
@@ -0,0 +1,179 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import arrow from 'components/LoginPage/images/arrow.svg'
+import slideImage1 from 'components/LoginPage/images/slide-3_img-1.svg';
+import slideImage2 from 'components/LoginPage/images/slide-3_img-2.svg';
+
+const Slide = styled.section`
+  padding-bottom: 0%;
+  text-align: center;
+  .note-example {
+    width: 61%;
+  }
+   .edit-example {
+    position: relative;
+    margin-top: -9%;
+  }
+  .bottomPosition {
+    margin-top: -65%;
```

Comment:
Why so big negative margin?
`.bottomPosition` is something that has `margin top -65%` and `margin bottom 59%`. Looks very weird, unreadable.

Reply:
Ok, I'll try to solve this in another way.